server/src/main/java/com/testsigma/controller/TestCasesController.java

```diff
@@ -158,7 +158,11 @@ public ResponseEntity<String> bulkMarkAsDelete(@RequestBody(required = false) Ma
         }
       }
     }
-    testCaseService.markAsDelete(validIds);
+    try {
```

Comment:
@nixonakash01 What is the need for a catch block here ? If the markAsDelete function fails, you will still be returning a 200 OK response here which is not expected

Reply:
The cloud block was implemented in cloud first. But its removed now. Ill remove it too.